lib/jobs/create-default-pollers.js

```diff
@@ -52,6 +52,7 @@ function createDefaultPollersJobFactory(
      * @param {String} obmName: OBM service name required to create pollers
      */
     CreateDefaultPollersJob.prototype._createWorkitem = function(nodeId, poller, obmName) {
+        var clonedPoller = _.cloneDeep(poller);
```

Comment:
It looks like the 'poller' here is just a 'pollerTemplate' , it could apply to different nodes, create the same type pollers in multiple nodes.

Reply:
fixed.
Use 'pollerTempete' as variable to instead of poller in create-default-poller.js